eclipse-solargraph-plugin/src/main/java/io/github/pyvesb/eclipse_solargraph/utils/LogHelper.java

```diff
@@ -22,14 +22,31 @@
 public class LogHelper {
 
 	private static final Bundle BUNDLE = FrameworkUtil.getBundle(LogHelper.class);
+	private static final String LOGNAME = BUNDLE.getSymbolicName();
```

Comment:
`getSymbolicName` seems to be doing a surprising amount of work underneath the hood, good to pull it up as a constant. 👍🏻